Gemfile

```diff
@@ -0,0 +1,50 @@
+source 'https://rubygems.org'
```

Comment:
- [ ] Kindly fix this error

![image](https://github.com/harshi0102/hello-rails-backend/assets/24703171/0f304cfc-5c69-41b7-8993-610146e10102)

I get this error when I run the frontend project in the console and I do not get any greeting message, but instead, I get an error message `Failed to fetch`.

Reply:
fix